app/views/channelsmodal.html

```diff
@@ -17,33 +17,34 @@ <h4 class="modal-title" id="addChannelLabel">
 
   <fieldset>
     <tabset justified="true">
-      <!-- ************* TAB FOR CHANNEL ACCESS CONTROL ************** -->
+      <!-- ************* TAB FOR CHANNEL BASIC INFO ************** -->
```

Comment:
Let's change the tab order:
* Basic Info
* Request Matching
* Routes
* Data Control
* User Access (renamed from Access Control)
* Alerts